plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -305,26 +305,33 @@ protected void renameTable(ConnectorSession session, Connection connection, Stri
         if (!remoteSchemaName.equals(newRemoteSchemaName)) {
             throw new TrinoException(NOT_SUPPORTED, "This connector does not support renaming tables across schemas");
         }
-
-        execute(connection, format(
-                "sp_rename %s, %s",
-                singleQuote(catalogName, remoteSchemaName, remoteTableName),
-                singleQuote(newRemoteTableName)));
+        String fullTableFromName = DOT_JOINER.join(
+                quoted(catalogName),
+                quoted(remoteSchemaName),
+                quoted(remoteTableName));
+
+        try (CallableStatement renameTable = connection.prepareCall("exec sp_rename ?, ?")) {
+            renameTable.setString(1, fullTableFromName);
+            renameTable.setString(2, newRemoteTableName);
```

Comment:
fullTableFromName -> oldTableName

Reply:
`oldTableFullName` (it's not just the table name)

Reply:
actually I didn't like `full` I've spent some time to understand that it's `full name of the table which has to be renamed`.
`oldTableFullName` - table is not old. Full name is old.

Reply:
tableOldFullName?

Reply:
To be honest I don't think there are any good names here - I'll just go with what exists already. 😄 We can rename when we come up with better name.